### PR TITLE
build: add CMake configuration for evread module

### PR DESCRIPTION
**Pull request by mkokryashkin**
This patch adds necessary cmake configurations for the <evread.lua> module, so it can be used later to implement human-readable error reporting in profile parsers.

Part of #9217

NO_DOC=LJ
NO_TEST=LJ
NO_CHANGELOG=build

### CODE REVIEW

**Review by Buristan — APPROVED** [review]
Hi, Maxim!
Thanks for the patch!
LGTM, except a general nit to all changes.
Please sort all new entries alphabetically.

**Comment by mkokryashkin** [comment]
> Hi, Maxim! Thanks for the patch! LGTM, except a general nit to all changes. Please sort all new entries alphabetically.

Thanks for the comment! Fixed for places where order is insignificant.

**Review by ligurio — COMMENTED** [review]
I would replace `LJ` with something more clear, for example "covered by LuaJIT tests":

```
NO_DOC=LJ
NO_TEST=LJ
```

File: src/lua/init.c [hunk 1]
```diff
@@ -229,6 +230,7 @@ static const char *lua_modules[] = {
 	"utils.avl", utils_avl_lua,
 	"utils.bufread", utils_bufread_lua,
 	"utils.symtab", utils_symtab_lua,
+	"utils.evread", utils_evread_lua,
```
Comment (ligurio) [inline src/lua/init.c:233]: Why not alphabetically sorted?

**Review by ligurio — APPROVED** [review]
Maxim, thanks for the patch! LGTM with minor comments.

File: src/lua/init.c (hunk 1 above)
Comment (mkokryashkin) [reply src/lua/init.c:233]: Because the symtab module is used by evread, hence it should go first. In other places order has no such meaning.

**Comment by mkokryashkin** [comment]
> I would replace `LJ` with something more clear, for example "covered by LuaJIT tests":
> 
> ```
> NO_DOC=LJ
> NO_TEST=LJ
> ```

Fixed, thanks

File: src/lua/init.c (hunk 1 above)
Comment (ligurio) [reply src/lua/init.c:233]: Thanks!

**Review by ligurio — APPROVED** [review]

**Review by ylobankov — APPROVED** [review]

**Pull request merged**